caluma/caluma_core/jexl.py

```diff
@@ -68,6 +70,29 @@ def __init__(self, *args, **kwargs):
         self.add_transform("debug", self._debug_transform)
         self._expr_stack = []
 
+        self.add_transform(
+            "mapby", lambda arr, key: [obj.get(key, None) for obj in arr]
+        )
+        self.add_binary_operator(
+            "intersects", 20, lambda left, right: any(x in right for x in left)
+        )
+
+        def _handle_type_error(func, *args):
+            try:
+                return func(*args)
+            except TypeError:
+                return None
+
+        self.add_transform("min", partial(_handle_type_error, min))
+        self.add_transform("max", partial(_handle_type_error, max))
+        self.add_transform("sum", partial(_handle_type_error, sum))
```

Comment:
Those should not return `None` but filter out the faulty values (non numbers)

Reply:
Also, `sum` is not implemented in JS yet - did you discuss this?

Reply:
`sum` is now implemented in JS. @sbor23 can you add `avg` as well?